cocinaborcelle/src/components/CardCompras.tsx

```diff
@@ -0,0 +1,49 @@
+
+import { useState } from 'react';
+import styles from "@/styles/CardCompras.module.css";
+
+type CardComprasProps = {
+    title?: string;
+    number: number;
+}
+
+export default function CardCompras({ title }: CardComprasProps) {
+    const [count, setCount] = useState(0);
+
+    const handleIncrement = () => {
+        setCount(count + 1);
```

Comment:
```suggestion
        // substituir código para uma chamada para API, alterando a quantidade da compra no banco de dados
        setCount(count + 1);
```

é bom deixar comentado como a pessoa pode melhorar esse componente, para quando integrar com a API